src/registrar/models/user.py

```diff
@@ -39,6 +51,19 @@ def __str__(self):
         else:
             return self.username
 
+    def block_user(self):
+        self.status = "ineligible"
```

Comment:
```suggestion
        self.status = self.INELIGIBLE
```
Ignore this if you think its possible the values of 'STATUS_CHOICES' and 'self.status' may drift, but as a general rule its good to have constants cascade down like this - keeps the codebase a bit more predictable (as a dev may, rarely, miss a field if editing manually) and it makes it a bit easier to do a mass change. Not really a huge thing though

Reply:
Agree with Zander on this as well